.docker/plugins/rbd/.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7.4.1708
+FROM centos:latest
```

Comment:
Hi @arm64b,

I'm not sure we want to pull the latest. Why make this change? I prefer that `FROM` statements refer to a specific version for purposes of reproducibility. @codenrhoden?

Reply:
Ah, this should be stale. I think we can close this PR now 🛩